cucumber/authenticators_jwt/features/authn_jwt.feature

```diff
@@ -41,6 +41,35 @@ Feature: JWT Authenticator - JWKs Basic sanity
     And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
     And I successfully set authn-jwt "token-app-property" variable to value "host"
 
+  Scenario: jwks-uri dynamically changed, 401 ERROR resolves 200 OK
+    Given I have a "variable" resource called "test-variable"
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I permit host "myapp" to "execute" it
+    And I successfully set authn-jwt "jwks-uri" variable to value "incorrect.com"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the audit log file
+    And I authenticate via authn-jwt with raw service ID
+    And the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
```

Comment:
I think its better here to do:
When I authenticate...
Then the HTTP...
And The following
You can have more than one action/response i think it will be better